lib/run-in-atom.coffee

```diff
@@ -36,17 +41,33 @@ module.exports =
   runCodeInScope: (code, scope, callback) ->
     switch scope
       when 'source.coffee', 'source.embedded.coffee'
+        vm.runInThisContext(console.clear()) if atom.config.get 'run-in-atom.clearConsoleBeforeRun'
```

Comment:
It might be worth trying this as prepending `console.clear();` to `code` and then only calling `runInThisContext` once. I'd imagine that would be more efficient.